target/linux/ar71xx/image/generic.mk

```diff
@@ -1,3 +1,20 @@
+define Device/OpenMesh
+  FILESYSTEMS := squashfs
+  DEVICE_PACKAGES := om-watchdog
+  KERNEL := kernel-bin | lzma | uImage lzma
+  IMAGES := sysupgrade.bin factory.bin
+  IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-rootfs | combined-ext-image
```

Comment:
append-metadata will not do anything when SUPPORTED_DEVICES is not defined. And defining them will "break" the image (at least it not anymore in the CE01 format). It is maybe the same reason why combined-image are also not created with append-metadata. in the last step. But that is just a wild guess.

I was also thinking about the image size check but it was already done in the ce image script. And this script had access to the actual firmware and kernel sizes at the same time (which is the important factor for the openmesh devices). But maybe the check could be added before the combined-ext-image is called - haven't tested that